packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,260 @@
+                <Stack.Item className={phoneInfoStep}>
+                  {conferencePhoneInfoList.map((phoneNumber, index) => (
+                    <Stack.Item key={index}>
+                      <Text className={phoneInfoTextStyle}>{formatPhoneNumberInfo(phoneNumber, strings)}</Text>
+                    </Stack.Item>
+                  ))}
+                </Stack.Item>
+              </Stack>
+              <Stack
+                horizontal
+                horizontalAlign="space-between"
+                verticalAlign="center"
+                className={phoneInfoInctructionLine}
+              >
+                {}
+                <Stack.Item style={{ display: 'flex' }}>
+                  <Stack horizontal>
+                    <Stack className={infoConnectionLinkStyle(theme)}></Stack>
+                    <Stack.Item className={phoneInfoIcon(theme)}>
+                      <Stack verticalAlign="center" horizontalAlign="center">
+                        <Icon
+                          iconName="DtmfDialpadButton"
+                          style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                        />
+                      </Stack>
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Text className={phoneInfoTextStyle}>{formatMeetingId(conferencePhoneInfoList[0].conferenceId)}</Text>
+              </Stack>
+              <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+                <Stack horizontal>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Icon iconName="PhoneInfoWait" style={{ color: theme.palette.themePrimary, padding: '8px' }} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack>
+            </Stack>
+          )}
+        </Modal>
+      }
+    </>
+  );
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  let enchantedPhoneNumber = phoneNumber;
+  if (!phoneNumber.startsWith('+')) {
+    enchantedPhoneNumber = `+${phoneNumber}`;
+  }
+  if (isPossiblePhoneNumber(enchantedPhoneNumber)) {
+    return parsePhoneNumber(enchantedPhoneNumber)?.formatInternational() || enchantedPhoneNumber;
+  }
+  return phoneNumber;
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumberInfo = (
+  phoneNumber: ConferencePhoneInfo | undefined,
```

Comment:
can this be called with an undefined phoneNumber? The UI would show a blank entry then